src/Analyzer.Reports/ConsoleReportWriter.cs

```diff
@@ -24,6 +35,10 @@ public void WriteResults(IEnumerable<Types.IEvaluation> evaluations, IFileInfo t
             {
                 fileMetadata += Environment.NewLine + $"Parameters File: {parametersFile}";
             }
+            if (configurationsFile != null)
+            {
+                fileMetadata += Environment.NewLine + $"Configurations File: {configurationsFile}";
```

Comment:
There's also a Console.WriteLine with this information in some other class, wouldn't this end up duplicated in the console?

Reply:
I don't think this should print the configuration file in the file metadata - it would be reprinted for every template analyzed.